### ISSUE TITLE
The react-scripts package provided by Create React App requires a dependency:    "babel-eslint": "9.0.0"

### ISSUE THREAD

**Issue by Chawki726** [eslint/eslint#11229]
There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "babel-eslint": "9.0.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of babel-eslint was detected higher up in the tree:

  C:\Users\chawki\node_modules\babel-eslint (version: 6.1.2)

Manually installing incompatible versions is known to cause hard-to-debug issues.

**Comment by JijoBose** [comment]
Hi @Chawki726 

create-react-app uses 9.0.0, 
You'll need to install the specific version 9.0.0
 
```
yarn remove babel-eslint
yarn add --dev babel-eslint@9.0.0
```

**Comment by not-an-aardvark** [comment]
Glad you figured it out, I'll go ahead and close this issue.

**Comment by dutradotdev** [comment]
@Chawki726 this worked for me man, tks!!

**Comment by sirajalam049** [comment]
Add 
`SKIP_PREFLIGHT_CHECK=true`
in your .env file

**Comment by Sshetty2** [comment]
I had accidently installed a dependencies higher up in my tree like the error message stated; on my desktop. The dependencies happened to include a this babel-eslint dependency that conflicted with the newer version that react-scripts required

**Comment by dkang933** [comment]
@Chawki726 worked like magic. thanks~
